src/registrar/views/domain_request.py

```diff
@@ -368,8 +382,7 @@ def db_check_for_unlocking_steps(self):
                 or self.domain_request.no_other_contacts_rationale is not None
             ),
             "additional_details": (
-                (self.domain_request.anything_else is not None and self.domain_request.has_cisa_representative)
-                or self.domain_request.is_policy_acknowledged is not None
+                (self.domain_request.has_anything_else_text and self.domain_request.has_cisa_representative)
```

Comment:
What is this fixing? It seems to be locking the "additional details" tab even after completing it if you entered both values as false.